stock_picking_declared_value/models/stock_picking.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2025 Nitrokey GmbH
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    # Add a computed field to show the total declared value of the picking
+    amount_total = fields.Monetary(
+        string="Total Declared Value",
+        compute="_compute_amount_total",
+        store=True,
+        help="Total declared value of the picking based on the values of the products",
+    )
+    currency_id = fields.Many2one(
+        "res.currency",
+        string="Currency",
+        related="sale_id.currency_id",
+        readonly=True,
+    )
+    declared_amount = fields.Float(
```

Comment:
Wouldn't it be better to rename it (maybe `declared_amount_percentage`) here and on `delivery.carrier` and use `declared_amount` as a computed field based on the percentage and the `amount_total`.

Here it's `declared_amount` but the module uses `declared_value` one wording should be picked.